retrofit2/src/main/java/com/linecorp/armeria/client/retrofit2/ArmeriaCallFactory.java

```diff
@@ -48,12 +53,28 @@
 import okhttp3.Response;
 import okio.Buffer;
 import okio.Timeout;
+import retrofit2.Invocation;
 
 /**
  * A {@link Factory} that creates a {@link Call} instance for {@link HttpClient}.
  */
 final class ArmeriaCallFactory implements Factory {
 
+    private static final AttributeKey<Invocation> RETROFIT_INVOCATION =
+            AttributeKey.valueOf(ArmeriaCallFactory.class, "RETROFIT_INVOCATION");
+
+    /**
+     * Retrieves a Retrofit {@link Invocation} associated with a {@link RequestLog}.
+     */
+    @Nullable
+    public static Invocation getInvocation(RequestLog log) {
```

Comment:
I think a user will not be able to access `Invocation` when he or she wants to write his or her own `MeterIdPrefixFunction`. How about reviving `InvocationUtil`?

Reply:
Couldn't think of that the user would use. Sorry about the mess. 😢

Reply:
I forget ArmeriaCallFactory is not public..0rz..